projects/core/src/occ/occ-models/occ-endpoints.model.ts

```diff
@@ -1,364 +1,369 @@
+export const DEFAULT_SCOPE = 'default';
+
+export interface OccEndpoint {
+  default?: string;
+  [scope: string]: string;
+}
+
+export interface ProductOccEndpoint extends OccEndpoint {
+  default?: string;
```

Comment:
This default was needed if this already extends OccEndpoint?

Reply:
Same with [scope:string]

Reply:
👍